package.json

```diff
@@ -51,7 +51,7 @@
     "sw-precache": "^5.2.0",
     "webpack": "^3.12.0",
     "webpack-cli": "^3.1.2",
-    "webpack-dev-server": "^3.1.11",
+    "webpack-dev-server": "3.0.0",
```

Comment:
Why is this change part of the diff?

Should we update to webpack 4 on master?

Reply:
This should had been committed to master (same as new `.ruby-version` file) but since we're merging soon didn't seem to make a difference.
Downgrade is because `webpack-dev-server` only works with `webpack 3`, I didn't want to do the upgrade now so I chose the easy path. Can work on upgrading to v4 after this is all done.